back-up-and-restore-data.md

```diff
@@ -194,9 +195,51 @@ INSERT INTO quotes (quote, characters, stardate, episode) VALUES
     ...
 ~~~
 
+### Dump table data as of a specific time
+
+In this example, we use the [`cockroach sql`](use-the-built-in-sql-client.html) command to create a table and execute two distinct inserts, returning the timestamp of the second insert. We then use the `cockroach dump` command to dump the table as of the timestamp of the second insert. Note that because the `--as-of` timestamp is exclusive, the dump does not include the second insert.
+
+~~~ shell
+$ cockroach sql --execute="CREATE TABLE db1.dump_test (id SERIAL PRIMARY KEY, name STRING)"
+~~~
+
+~~~ shell
+CREATE TABLE
+~~~
+
+~~~ shell
+$ cockroach sql --execute="INSERT INTO db1.dump_test (name) VALUES ('foo')"
+~~~
+
+~~~ shell
+INSERT 1
+~~~
+
+~~~ shell
+$ cockroach sql --execute="INSERT INTO db1.dump_test (name) VALUES ('bar') RETURNING now()"
```

Comment:
This example is misleading because `now()` returns microsecond precision, but internally there was nanosecond precision. The nanos are being truncated, which means the timestamp printed isn't quite the "database timestamp" at which that insert occurred. See:

```
root@:26257/d> CREATE TABLE db1.dump_test (id SERIAL PRIMARY KEY, name STRING);
CREATE TABLE
root@:26257/d> INSERT INTO db1.dump_test (name) VALUES ('foo')
            -> ;
INSERT 1
root@:26257/d> INSERT INTO db1.dump_test (name) VALUES ('bar') RETURNING cluster_logical_timestamp();
+--------------------------------+
|  cluster_logical_timestamp()   |
+--------------------------------+
| 1488910193832885496.0000000000 |
+--------------------------------+
(1 row)
root@:26257/d> select * from db1.dump_test as of system time 1488910193832885496.0000000000;
+--------------------+------+
|         id         | name |
+--------------------+------+
| 225574025592307713 | foo  |
| 225574236432171009 | bar  |
+--------------------+------+
(2 rows)
root@:26257/d> SELECT * FROM db1.dump_test AS OF SYSTEM TIME 1488910193832885495.0000000000;
+--------------------+------+
|         id         | name |
+--------------------+------+
| 225574025592307713 | foo  |
+--------------------+------+
(1 row)
```

However `cockroach dump --as-of` doesn't support the `cluster_logical_timestamp()` return format, so we can't have that in our example. (--as-of could be improved to support that format if we want, though.)